decode_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/testing/protocmp"
 	"google.golang.org/protobuf/types/known/durationpb"
+	"gotest.tools/v3/assert"
```

Comment:
wrong assert library? Should be github.com/stretchr/testify/assert, right?

Reply:
oh wow